src/ExtensionRoot.ts

```diff
@@ -94,7 +93,7 @@ export class ExtensionRoot {
         const experimentationService = await createExperimentationServiceAsync(context, new ExperimentationTelemetry(this._logger));
 
         const expectedCLIVersion: string = await VersionUtility.getExpectedCliVersionAsync(packageJsonContent);
-        const binariesUtility: IBinariesUtility = BinariesManager.getBinariesUtility(this._logger, context, commandEnvironmentVariables, accountContextManager, expectedCLIVersion);
```

Comment:
💡 Thank you for this, I believe you have tested this locally, so we are getting rid of one aka old `BinariesManager.getBinariesUtility` with choice to `v2` ? For rest of deleted files have we made sure there is not other file referencing this funct? please. thanks.

Reply:
Thanks for the review, yes I have tested locally and made sure non of the files are referencing this func.